### PR TITLE
Update security page

### PR DESCRIPTION
**Pull request by koss-lebedev**

### CODE REVIEW

File: project-guidelines/quality-output/security/README.md [hunk 1]
```diff
@@ -1,12 +1,83 @@
...
+- Exclude admin and private URLs from indexing and crawling (using robots.txt)
+- Don’t use iterable URLs (e.g. prefer `/users/[guid]` or `/users/[handle]` instead of `/users/[incremented-integer]`)
+- Remove unused functionality instead of hiding it (with `display: none`, for instance)
+- Use DDOS mitigation via a global caching proxy service like CloudFlare
+- For apps containing sensitive data, consider implementing auto session termination after a period of inactivity or require addition password check for destructive actions
+- Set `Cache-Control: no-store, max-age=0` for non-public/non-static content to make sure that the sensitive user-specific data doesn’t remain cached on disk after a logout
+- Add [subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) checks if loading your JavaScript libraries from a third party CDN
 
-Use amazing [https://observatory.mozilla.org/](https://observatory.mozilla.org/)
+## Server secrets
```
Comment (michalhonc) [inline project-guidelines/quality-output/security/README.md:18]: @koss-lebedev How about mentioning [@strv/shh](https://github.com/strvcom/shh)?

Comment (lucasconstantino) [reply project-guidelines/quality-output/security/README.md:18]: @koss-lebedev: something like this?

> Alternatively to not versioning `.env` files, you can encrypt & version them using [git-crypt](https://github.com/AGWA/git-crypt) – but that would assume anyone with access to the repository should ultimately have access to all secrets, which isn't always the case.

File: project-guidelines/quality-output/security/README.md [hunk 2]
```diff
@@ -1,12 +1,83 @@
...
+Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
+```
+
+- Enable [X-XSS-Protection](https://www.owasp.org/index.php/OWASP_Secure_Headers_Project#X-XSS-Protection) mode to mitigate XSS attacks
+
+```bash
+X-XSS-Protection: 1; mode=block
+```
+
+## Github
```
Comment (danbalarin) [inline project-guidelines/quality-output/security/README.md:79]: I'd mention here that if you are having some Github Actions in use, it's safer to have separate "GH application" with limited scope doing those jobs. Also added bonus is that you can permit pushes to certain branches only for that account.

Example here: https://github.com/strvcom/xx-aviation-frontend/blob/develop/.github/workflows/xx-merge-reusable.yaml

File: project-guidelines/quality-output/security/README.md [hunk 3]
```diff
@@ -1,12 +1,83 @@
...
 
-Be familiar with [OWASP Top Ten
-](https://owasp.org/www-project-top-ten/)
+- Don’t hardcode API keys and tokens, store them in the `.env` file instead
+- Don’t commit `.env` file to the repository
+- Don’t expose secret tokens from `.env` file to the client (make sure that you only access them in the server code, e.g. inside React server components or `getServerSideProps`)
+
+## Client secrets
+
+- Store session IDs and auth tokens in `Secure` `HttpOnly` cookies, not `localStorage` or `sessionStorage` (also SameSite=Lax)
```
Comment (danielkraus) [inline project-guidelines/quality-output/security/README.md:26]:
```suggestion
- Store session IDs and auth tokens in `Secure` `HttpOnly` cookies, not `localStorage` or `sessionStorage` (also [SameSite=Lax](https://web.dev/samesite-cookies-explained/#explicitly-state-cookie-usage-with-the-samesite-attribute))
```

**Review by danielkraus — APPROVED** [review]
@koss-lebedev I like it! Good to go.

**Review by lucasconstantino — APPROVED** [review]
❤️

**Pull request merged**